packages/integration-tests/index.js

```diff
@@ -564,6 +563,46 @@ async function deleteRules(stackName, bucketName, rules, postfix) {
   return rules.length;
 }
 
+/**
+ * Delete a rule's Kinesis Event Source Mappings
+ *
+ * @param {Object} rule - a Rule record as returned by the Rules api
+ * @param {string} rule.name
+ * @param {Object} rule.rule
+ * @param {string} rule.rule.arn
+ * @param {string} rule.rule.logEventArn
+ * @returns {Promise<unknown[]>} - Event Source Map deletion results
+ */
+async function deleteRuleResources(rule) {
+  const kinesisSourceEvents = [
+    {
+      name: getRequiredEnvVar('messageConsumer'),
+      eventType: 'arn',
+      type: {
+        arn: rule.rule.arn,
```

Comment:
:can-of-worms: Is this an ARN or a UUID? The function that's consuming this value seems to think it's a UUID, and the AWS docs seem to agree.